package.json

```diff
@@ -29,7 +29,8 @@
     "react-dom": "0.14.7",
     "react-redux": "4.4.1",
     "redux": "3.3.1",
-    "redux-logger": "2.6.1"
+    "redux-logger": "2.6.1",
+    "sqlite3": "^3.1.3"
```

Comment:
Other than one exception, all other modules have versions explicitly specified, any reason not to do that here?